### PR TITLE
Polish doc style of Google colab

### PR DESCRIPTION
**Pull request by hkvision**
https://analytics-zoo.github.io/master/#ProgrammingGuide/run-notebook-colab/
Some contents not beautifully rendered.

Need more test when I go back to office. Put this PR here first.

### CODE REVIEW

File: docs/docs/ProgrammingGuide/run-notebook-colab.md [hunk 1]
```diff
@@ -27,7 +27,9 @@ You can add the following command on your colab file to install the analytics-zo
 !pip install analytics-zoo
 ```
 
-Begin your code. Call `init_nncontext()` that will create a SparkContext with optimized performance configuration and initialize the BigDL engine.
+**Begin your code**
+
+Call `init_nncontext()` that will create a SparkContext with optimized performance configuration and initialize the BigDL engine.
```
Comment (jason-dai) [inline docs/docs/ProgrammingGuide/run-notebook-colab.md]: remove "and initialize the BigDL engine"

File: docs/docs/ProgrammingGuide/run-notebook-colab.md [hunk 2]
```diff
@@ -2,9 +2,9 @@ With [Google Colaboratory](https://colab.research.google.com/), we can easily se
 
 First, create or load a notebook file in colaboratory. Then, prepare the environment. Only you need to prepare is installing JDK and Analytics Zoo. As installing analytics-zoo from pip will automatically install pyspark, you are recommended to not install pyspark anymore.
```
Comment (jason-dai) [inline docs/docs/ProgrammingGuide/run-notebook-colab.md]: "You only need to install JDK and Analytics Zoo"

"you are recommended not to install pyspark "

File: docs/docs/ProgrammingGuide/run-notebook-colab.md [hunk 3]
```diff
@@ -48,8 +50,8 @@ If you would like to open Analytics Zoo Notebook in a GitHub repo directly, the
...
 
-  For example, change the URL of Analytics Zoo tutorial https://github.com/intel-analytics/zoo-tutorials/blob/master/keras/2.1-a-first-look-at-a-neural-network.ipynb to https://colab.research.google.com/github/intel-analytics/zoo-tutorials/blob/master/keras/2.1-a-first-look-at-a-neural-network.ipynb 
+  For example, change the URL of Analytics Zoo tutorial [https://github.com/intel-analytics/zoo-tutorials/blob/master/keras/2.1-a-first-look-at-a-neural-network.ipynb] to [https://colab.research.google.com/github/intel-analytics/zoo-tutorials/blob/master/keras/2.1-a-first-look-at-a-neural-network.ipynb] 
 
-Then, prepare the environment of Java8 and Analytics Zoo as instructions above at the beginning of the colab notebook. If you would like to save changes, you can make a copy to drive and run it within the instructions.
+Then, prepare the environment of Java8 and Analytics Zoo as instructions [above](#prepare-environment) at the beginning of the colab notebook. If you would like to save changes, you can make a copy to drive and run it within the instructions.
```
Comment (jason-dai) [inline docs/docs/ProgrammingGuide/run-notebook-colab.md]: `as described above](#prepare-environment)`

**Comment by hkvision** [comment]
Doc jenkins passed: http://10.239.47.210:18888/job/ZOO-PR-Doc-Generation-Test/246/

**Comment by hkvision** [comment]
Manually review the updated style on desktop.

**Pull request merged**